PayPalNativePayments/src/test/java/com/paypal/android/paypalnativepayments/PayPalNativeCheckoutClientTest.kt

```diff
@@ -333,7 +334,7 @@ class PayPalNativeCheckoutClientTest {
         val shippingData = mockk<ShippingChangeData>(relaxed = true)
 
         every { shippingData.shippingChangeType } returns ShippingChangeType.ADDRESS_CHANGE
-        every { shippingData.shippingAddress } returns Address(countryCode = mockCountryCode)
```

Comment:
This may have caused detekt to fail with an `unused import` message.

Reply:
yeah just saw that one